demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -541,6 +563,25 @@ def _is_approved_tags(self) -> bool:
                 return False
         return True
 
+    def _is_right_version(self):
+        """Checks whether the currentVersion field in the pack metadata match the version of the latest release note.
+
+        Return:
+             bool: True if the versions are match, otherwise False
+        """
+        metadata_file_path = self._get_pack_file_path(self.pack_meta_file)
+        current_version = self.metadata_content.get('currentVersion', '0.0.0')
+        rn_version = self._get_pack_latest_rn_version()
+        if not rn_version and current_version == '1.0.0':
+            return True
+        if not rn_version:
+            self._add_error(Errors.missing_release_notes_for_pack(self.pack), self.pack)
+            return False
+        if LooseVersion(rn_version) != LooseVersion(current_version):
```

Comment:
LooseVersion is deprecated. Use [packaging.version.parse](https://packaging.pypa.io/en/latest/version.html#packaging.version.parse)